corehq/apps/domain/static/domain/js/case_search.js

```diff
@@ -13,22 +13,22 @@ hqDefine('domain/js/case_search', [
 ) {
     var module = {};
 
-    var Property = function (name) {
+    var property = function (name) {
```

Comment:
Nitpick, but I'd call this `propertyModel`, it's confusing to have a `property` function and then a `property` variable a few lines below.